lib/ratfun1.gi

```diff
@@ -42,9 +42,9 @@ local f,typ,lc;
     typ := rfam!.threeLaurentPolynomialTypes[1];
   fi;
   
-  # slightly better to do this after the Length has been determined 
+  # slightly better to do this after the Length has been determined
   if IsFFECollection(coeff) and IS_PLIST_REP(coeff) then
-    ConvertToVectorRep(coeff);
+    coeff := ImmutableVector(DefaultRing(coeff), coeff);
```

Comment:
Warning: in `lib/ratfun1.g`, there is actually one other diff, which I ignored: in line 55f, this:
```gap
   f := rec(IndeterminateNumberOfUnivariateRationalFunction:=inum,
           CoefficientsOfLaurentPolynomial:=Immutable([coeff,val]));
```
is replaced by
```gap
   f := rec(IndeterminateNumberOfUnivariateRationalFunction:=inum,
           CoefficientsOfLaurentPolynomial:=MakeReadOnlyObj(CopyRegion([coeff,val])));
```
However, I could not find any example where this made a difference. Given that `[coeff,val]` is a plain list, and `coeff` usually is either a plain list, or a compressed vector, and `val` is a ring element, I kind of believe that using `Immutable` here should always be fine -- otherwise, we'd get problems in many other places. But then, somebody originally made this change, perhaps to fix some specific problem?